### PR TITLE
all: Remove ck8sdash

### PR DESCRIPTION
**Pull request by OlleLarsson**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
#240 

**Special notes for reviewer**:

**Checklist:**

- [ ] Added relevant notes to [WIP-CHANGELOG.md](https://github.com/elastisys/compliantkubernetes-apps/blob/main/WIP-CHANGELOG.md)
- [ ] Proper commit message prefix on all commits
- [ ] Updated the [public facing documentation](https://github.com/elastisys/compliantkubernetes)
- Is this changeset backwards compatible for existing clusters? Applying:
    - [ ] is completely transparent, will not impact the workload in any way.
    - [ ] requires running a migration script.
    - [ ] will create noticeable cluster degradation.
          E.g. logs or metrics are not being collected or Kubernetes API server
          will not be responding while upgrading.
    - [ ] requires draining and/or replacing nodes.
    - [ ] will change any APIs.
          E.g. removes or changes any CK8S config options or Kubernetes APIs.
    - [ ] will break the cluster.
          I.e. full cluster migration is required.

### CODE REVIEW

File: helmfile/01-applications.yaml [hunk 1]
```diff
@@ -106,26 +106,6 @@ releases:
...
 {{ end }}
 
-
-- name: ck8sdash
-  namespace: ck8sdash
-  labels:
-    app: ck8sdash
-  chart: ./charts/ck8sdash
-  version: 0.3.1
-  installed: {{ .Values.ck8sdash.enabled }}
```
Comment (lentzi90) [inline helmfile/01-applications.yaml:116]: Should we keep this and hard code to `installed: false` to make it auto removed from all environments?

**Review by lentzi90 — COMMENTED** [review]
I see red, I like :slightly_smiling_face:

File: helmfile/01-applications.yaml (hunk 1 above)
Comment (OlleLarsson) [reply helmfile/01-applications.yaml:116]: Maybe, I wasn't sure whether to do that or write a script that removes everything including the ck8sdash namespace (but maybe that is installed from the rbac chart in most cases).

Comment (lentzi90) [reply helmfile/01-applications.yaml:116]: Right, whatever is easier works for me. :slightly_smiling_face:

**Comment by viktor-f** [comment]
ck8s dash had some credentials in grafana and/or kibana, right? If so then maybe we should add something in the release/upgrade notes about removing them.

File: helmfile/01-applications.yaml (hunk 1 above)
Comment (OlleLarsson) [reply helmfile/01-applications.yaml:116]: I went with your solution.

**Comment by OlleLarsson** [comment]
> ck8s dash had some credentials in grafana and/or kibana, right? If so then maybe we should add something in the release/upgrade notes about removing them.

Yes, there were some api keys in grafana. I've added removal instructions in the upgrade guide.

**Review by lentzi90 — APPROVED** [review]
LGTM!

Have you tested that helmfile can remove ck8sdash also with the chart files already gone? Do you need any help testing?

**Review by viktor-f — APPROVED** [review]
LGTM!

**Comment by OlleLarsson** [comment]
> LGTM!
> 
> Have you tested that helmfile can remove ck8sdash also with the chart files already gone? Do you need any help testing?

Yes, I've tested (I suppose it only needs to know the name of the release in order to remove it)

**Pull request merged**